### ISSUE TITLE
is bulk_origin_asn using the netcat bulk function?

### ISSUE THREAD

**Issue by randomgambit** [hrbrmstr/cymruservices#1]
Hello @hrbrmstr 

another great package from you! I would like to use the `bulk_origin_asn` for all the current 50k ASN availables. However,  I see that on `teamcymru` the warning:

> WARNING: IPs that are seen abusing the whois server with large numbers of individual queries instead of using the bulk netcat interface will be null routed. If at all possible you should consider using the DNS based query interface since it is much more efficient for individual queries. The netcat interface should be used for groups of IP lists at a time in one single TCP query.

So I just wanted to make sure that the `bulk_origin_asn` does that so that I wont be banned.

Thanks again!!

**Comment by hrbrmstr** [comment]
Yep. All the "bulk" interfaces use the WHOIS (i.e. netcat) interface.

NOTE: maxmind free ASN dbs can provide much of this data locally (i.e. w/o calling out to teamcymru)

**Comment by randomgambit** [comment]
got it, thanks harbour master :)

**Comment by randomgambit** [comment]
by the way, do you know how much is too much even for the `netcat` interface? can I pull the info for all the 65k ASNs? (yes I know there is the maxmind ASN but it does not contain the date of allocation).

thanks master!

**Comment by hrbrmstr** [comment]
Wow. Missed this. It's been a rly "interesting" summer.

You can do 50K at a time. However, I'd encourage you to also look at using `rgeolocate` and MaxMind's free ASN database with it. It's not perfect, but it'll likely get you what you need locally and then any `NA`s can be fed to cymru.
